File: 1/4.0/4.cpp
#include <iostream>
#include <Windows.h>

int main()
{
    SetConsoleCP(1251);
    SetConsoleOutputCP(1251);

    float result = 0.0;

    int n;

    while (true)
    {
        std::cout << "Введіть число: ";
        std::cin >> n;

        if (std::cin.good())
        {

            for (int k = 1; k <= n; k++)
            {
                result += 1 / (k * pow((2 * k + 1), 2));
            }

            std::cout << "Отриманий результат: " << result << std::endl;
            break;
        }
        else
        {
            std::cin.clear();
            std::cin.ignore(32767, '\n');
            std::cout << "Введіть саме число!" << std::endl;
            
        }
    }
}

File: 1/4.0/4.py

result = 0.0

while(True):

    try:
        n = int(input("Введіть число n: "))

        for k in range(1, n + 1):
            result += 1 / (k * pow((2 * k + 1), 2))

        print(f"Результат: {result}")
        break

    except ValueError:
        print("Введіть саме число!")




